core/tasks.md

```diff
@@ -27,9 +97,17 @@
     - require in index.js and console the output by executing it there.
 ```js
 // math.js
-const pie = 3.14;
-function sum() {}
-function multiply() {}
+
+  const pie = 3.14;
+  function sum() {
+    console.log(pie+pie);
+  }
+  function multiply() {
+    console.log(pie*pie);
+  }
+module.exports.pie = pie;
+module.exports.sum = sum();
+module.exports.multiply = multiply();
```

Comment:
You cannot export it one by one using module.exports, it will override previous exports and only last export i.e. multiply will be exported.
```js
module.exports = {
  pie, sum, multiply
}
```
Another thing is don't call the function when exporting it like 
```js
module.exports = {
  multiply: multiply()  // bad practice
}
```
you call the function when you use it in imported files.